templates/post-install-job.yaml

```diff
@@ -40,8 +40,8 @@ spec:
             driver: azure/kv
             options:
               keyvaultname: {{ .Values.keyvault.name }}
-              keyvaultobjectnames: neuvector-admin-password;neuvector-new-admin-password;{{ .Values.keyvault.licensececretname }}
-              keyvaultobjecttypes: secret;secret;secret
+              keyvaultobjectnames: neuvector-admin-password;neuvector-new-admin-password;{{ .Values.keyvault.licensececretname }};neuvector-slack-webhook
```

Comment:
Is the neuvector slack webhook any different to our regular one?
(just worried about more manual steps to setup an environment i.e. more secrets to add and more documentation to maintain)

Reply:
Slack webhooks are linked to a specific channel as far as I know and my idea is that this stuff should go to its own channel. In general I agree with you though.